### PR TITLE
GEODE-9042: Update docker-based scripts to build UG

### PR DESCRIPTION
**Pull request by alb3rtobr**
This PR adapts the docker-based scripts to build the user guide to the new bookbinder version.
I have refactored them, now only one script is necessary, and the build process is faster after the Docker image is generated by first time.
I also changed geode-book readme, to associate this script with the idea of "previewing the user guide" instead of "building the user guide".

### CODE REVIEW

**Review by dickcav — APPROVED** [review]

**Review by davebarnes97 — CHANGES_REQUESTED** [review]
Excellent work - the single-script approach is a user-friendly improvement, and it worked perfectly for me on the first try.
Consolidating the instructions in a single README also improves the user experience. I'd like to request some edits to the README before this PR is merged. I cannot edit this PR directly, so I have attached my revisions to this comment.
My changes to the README include:

- Format adjustments compatible with 'MacDown'
- Addition of `cd` commands in some example code
- Removing most of the section on publishing to the website, referring the reader, instead, to the geode-site README file for those details.
Good work, @alb3rtobr, and thank you!
[README-db.md.zip](https://github.com/apache/geode/files/6555476/README-db.md.zip)

**Comment by alb3rtobr** [comment]
Thanks @davebarnes97 , changes applied. I have reworded the third point of the instructions, it looked redundant to me. Is it ok for you?

File: dev-tools/docker/docs/Dockerfile
Comment (moleske) [inline dev-tools/docker/docs/Dockerfile:17]: Just as an fyi, the latest (and last) release of 2.5.x is 2.5.9.  Of course this being ruby, I would not be shocked to find out exactly 2.5.3 is needed for this to all work

**Review by davebarnes97 — APPROVED** [review]
LGTM

File: dev-tools/docker/docs/Dockerfile
Comment (davebarnes97) [reply dev-tools/docker/docs/Dockerfile:17]: Does Docker allow you to specify greater-than-or-equal (>=) with a version number? That would be ideal. If not, 2.5.3 is a good working minumum version with the current toolset.

Comment (moleske) [reply dev-tools/docker/docs/Dockerfile:17]: You can use `FROM ruby:2.5` to get the latest ruby 2.5 version.  I wouldn't block merging the PR on this.  Ruby 2.5 is end of life anyway the real answer is use some tools that work with later ruby versions

Comment (alb3rtobr) [reply dev-tools/docker/docs/Dockerfile]: I tried using `ruby:2.5` as base image. As it works fine, I have updated the Dockerfile. Anyway, as @moleske pointed out, Ruby 2.5 is dead. Is anyone working actively on making bookbinder to work on Ruby 2.7?

**Pull request merged**
